src/plugins/data_table/filter_view.js

```diff
@@ -107,20 +107,6 @@ var DataTableFilter = (function(options) {
         <% }) %>\
       ', null, DEFAULT_JST_DELIMS),
 
-    afterRender: function () {
-      var listClass;
-
-      if (this.filter.options.length > 30) {
-        listClass = "triple";
-      } else if (this.filter.options.length > 15) {
-        listClass = "double";
-      } else {
-        listClass = "single";
-      }
-
-      this.$el.addClass(listClass);
```

Comment:
Am I missing something where this line couldn't become `this.$('.filterMenu').addClass(listClass)` and address the issue?